express/blocks/fullscreen-marquee/fullscreen-marquee.css

```diff
@@ -0,0 +1,282 @@
+.fullscreen-marquee {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.section > .fullscreen-marquee {
+  padding: 16px;
+  max-width: none;
+}
+
+.fullscreen-marquee.has-background {
+  padding-bottom: 160px;
+}
+
+.fullscreen-marquee .fullscreen-marquee-heading h1 {
+  font-size: var(--heading-font-size-l);
+  text-align: left;
+  margin: 0;
+}
+
+.fullscreen-marquee .fullscreen-marquee-heading p {
+  justify-content: center;
+  max-width: 680px;
+  margin: 10px 0;
+  font-size: var(--body-font-size-m);
+  text-align: left;
+}
+
+.fullscreen-marquee-heading h1.budoux {
```

Comment:
the class budoux used to be added to all header elements, but is no longer added in milo 
You'll need to check if you're on a japanese page in the js code and then add this class to your headers , otherwise the headers don't look right on japanese pages. 
see here 
- https://www.adobe.com/jp/express/create/story/instagram
- https://mwpw-143361--express-milo--adobecom.hlx.page/jp/express/create/story/instagram